pages/apis/graphql/graphql_cookbook.md

```diff
@@ -906,3 +906,25 @@ mutation UpdateTeamPipelineReadonly {
   }
 }
 ```
+
+## Test Analytics
+
+### Get number of builds between 2 dates 
+There is a correlation between number of builds and job minutes used. Though this query won't get you how many job minutes have been used, but it will give an idea of whether or not there is an increase in the number of builds, which will cause more job minutes to be used.
+
+```graphql
+query PipelineBuildCountForBranchQuery {
```

Comment:
I think this is a copy/paste hangover, so I think it should be updated. Perhaps:

```suggestion
query PipelineBuildCountForPeriod {
```